lib/grandpa/message_handler.go

```diff
@@ -414,11 +470,20 @@ func (s *Service) VerifyBlockJustification(hash common.Hash, justification []byt
 		"sig count", len(fj.Commit.Precommits),
 	)
 
-	if len(fj.Commit.Precommits) < (2 * len(auths) / 3) {
-		return ErrMinVotesNotMet
```

Comment:
you can leave this check in, as it will quickly check if there aren't enough precommits in the justification to ever be valid

Reply:
Done!